templates/ubuntu18/common/base.dockerfile.j2

```diff
@@ -200,6 +208,10 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends ${DEPS} && \
     rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates
+RUN curl https://raw.githubusercontent.com/openvinotoolkit/openvino/master/scripts/install_dependencies/install_openvino_dependencies.sh -o ${INTEL_OPENVINO_DIR}/install_dependencies/install_openvino_dependencies.sh && \
```

Comment:
Refering to master, remove before release or we will end up with outdated links in tagged commits